### DISCUSSION TITLE
How to handle JS from external apps?

### DISCUSSION THREAD

**Discussion by Teunel** [hayes0724/shopify-packer#113] [Q&A]
I use the app Product Filter & Search from Boost, they have a manual installation I like to use. 

The app includes some .js files I upload to my assets folder, that get included in theme.liquid. 

What is the recommended way to make changes to these files when developing a theme? The watch command does not upload changes I make to these files.

Is there a better way to handle files like these?

**Comment by molotow11** [comment]
Hello, Teunel.

Watch command working fine for me also for files inside assets folder, so it should be also synchronized. Try to check if you are working inside "src" folder.
